code/go/0chain.net/smartcontract/storagesc/alloation2_test.go

```diff
@@ -322,7 +322,8 @@ func TestFinalizeAllocation(t *testing.T) {
 
 	t.Run(ErrFinalizedTooSoon, func(t *testing.T) {
 		var allocationExpired = allocation
-		allocationExpired.Expiration = now - toSeconds(getMaxChallengeCompletionTime()) + 1
+		var conf = Config{}
+		allocationExpired.Expiration = now - toSeconds(conf.MaxChallengeCompletionTime) + 1
```

Comment:
You don't need to create a Config object here, you just need to set `allocationExpired.Expiration` to any number that guarantees that it can't be finalised yet because there might be active challenges. So to finalize an allocation, the allocation must have expired more than challenge completion time ago and guaranteing all challenges are completed.

Looking at the logic of the unit tests I think we delete  [blobberYaml.MaxChallengeCompletionTime](https://github.com/0chain/0chain/blob/d7321ef7a912eac8763e70eb6de034f69de6fadf/code/go/0chain.net/smartcontract/storagesc/alloation2_test.go#L251) and use [scYaml.MaxChallengeCompletionTime](https://github.com/0chain/0chain/blob/d7321ef7a912eac8763e70eb6de034f69de6fadf/code/go/0chain.net/smartcontract/storagesc/alloation2_test.go#L243)